pkgs/kubevirt/kubevirt/registry.yaml

```diff
@@ -0,0 +1,15 @@
+packages:
+  - type: github_release
+    repo_owner: kubevirt
+    repo_name: kubevirt
+    description: Kubernetes Virtualization API and runtime in order to define and manage virtual machines
+    asset: virtctl-{{.Version}}-{{.OS}}-{{.Arch}}
+    format: raw
+    replacements:
+      win: windows
+    files:
+      - name: virtctl
+    supported_envs:
+      - darwin
+      - linux
+      - windows/amd64
```

Comment:
Please add a newline at the end of files.

<img width="260" alt="image" src="https://github.com/aquaproj/aqua-registry/assets/13323303/31c05ff8-307e-4d2c-a1a0-37a4b549917f">